src/main/java/custom_exceptions/UserHandlingException.java

```diff
@@ -0,0 +1,7 @@
+package custom_exceptions;
+
+public class UserHandlingException extends Exception{
```

Comment:
`UserHandlingException` can be a packagename and specific exceptions can be added to the package.